modules/RTC/TraceablePeerConnection.js

```diff
@@ -813,18 +813,27 @@ TraceablePeerConnection.prototype.getTargetVideoBitrates = function() {
 };
 
 /**
- * Tries to find {@link JitsiTrack} for given SSRC number. It will search both
- * local and remote tracks bound to this instance.
+ * Tries to find {@link JitsiTrack} for given SSRC number. It will search both local and remote tracks bound to this
+ * instance.
  * @param {number} ssrc
+ * @param {isPrimary} boolean - Whether the SSRC passed is a primary SSRC of a source.
```

Comment:
It's not clear to me why we need the primary flag here. Wouldn't check the check always succeed if we always go throught he else branch?

Reply:
Before this change, this function was returning a track only if the passed SSRC was the primary SSRC and therefore stats were being processed only for the primary SSRC. If we modify this function to return the track for all the SSRCs in all the cases, this would result in extra processing. Let me check how bad it is going to be if we return the track for all SSRCs.